src/hw4.py

```diff
@@ -0,0 +1,59 @@
+# 1 Создайте декоратор, который хранит в себе результаты вызовов функции
+# и время получения результата за время запуска программы.
+import time
+
+global reclist
+reclist = []
+
+
+def eto_moy_decorator(func):
+    def obertochka(number):
+        print('Входной параметр = ', number + 1)
+        starttime = time.time()
+        func(number)
+        endtime = time.time()
+        result_of_call = 'Итог = ' + str(func(number)) + '  ' + \
```

Comment:
we call this func twice. what would happen if it contains random and returns different results?